index.js

```diff
@@ -1182,20 +1189,37 @@ module.exports = function (log, indexesPath) {
     return a.timestamp > b.timestamp
   }
 
-  function sortedByTimestamp(bitset, descending) {
+  function compareSeqAscending(a, b) {
+    return b.seq > a.seq
+  }
+
+  function compareSeqDescending(a, b) {
+    return a.seq > b.seq
+  }
+
+  function sortedBy(bitset, descending, sortBy) {
     updateCacheWithLog()
     const order = descending ? 'descending' : 'ascending'
+    const sortedCache = sortBy === 'arrival' ? sortedSeqCache : sortedTSCache
+    const comparer =
```

Comment:
The linter butchered this long line. It was originally like this:

```
    const comparer = sortBy === 'arrival' ?
          descending ? compareSeqDescending : compareSeqAscending :
          descending ? compareDescending : compareAscending
```

Reply:
It's fine, I know the feeling, let's let the linter own styling on our behalf